fp: rating and chars_count

7R41N33 opened the pull request.

artempartos commented on test/7r41n33/fp/solution.rb:
```diff
@@ -4,12 +4,22 @@ class << self
-      def rating(_films, _year)
-        0
+      def rating(films, year)
+        film_count = 0
```
считать количество в переменной не серьезно.
Попробуй отфильтровать те, которые надо считать  при помощи map.
Так у тебя будет и количество нужных тебе, и на них же можно сделать свертку при помощи reduce

7R41N33 replied:
Переделал

7R41N33 committed:
fp: rating and chars_count

Test result fixed

Tests fixed

rubocop refactoring

rubocop refactoring

fp: refactoring rating
